deep/views.py

```diff
@@ -42,6 +45,9 @@ def get_frontend_url(path=''):
         path=path,
     )
 
+def graphql_docs(request):
+    html = graphdoc.to_doc(GraphQLView().schema)
```

Comment:
To fix the __proxy__ type issue.
```suggestion
    html = graphdoc.to_doc(str(CustomGraphQLView().schema))
```